docs/android/README.md

```diff
@@ -56,7 +56,8 @@ Converted subprojects:
 
 Let’s take a look at a `build.gradle.dcl` file which replaces the common `build.gradle` or `build.gradle.kts` files:
 
-core/common/build.gradle.dcl
+_core/common/build.gradle.dcl_
```

Comment:
❗ The example below this can be updated to remove the entire `buildTypes` block, it no longer needs to be explicit.

Reply:
Let's do in a follow-uo